eli5/lightning.py

```diff
@@ -28,7 +27,6 @@ def explain_weights_lightning(estimator, vec=None, top=20, target_names=None,
     )
 
 
-@explain_prediction.register(BaseEstimator)
```

Comment:
I think it needs a fix similar to sklearn's, becaue BaseEstimator here is not sklearn's BaseEstimator